modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -28,11 +28,7 @@ export default class TileLayer extends CompositeLayer {
 
   updateState({props, oldProps, context, changeFlags}) {
     let {tileCache} = this.state;
-    if (
-      !tileCache ||
-      (changeFlags.updateTriggersChanged &&
-        (changeFlags.updateTriggersChanged.all || changeFlags.updateTriggersChanged.getTileData))
-    ) {
+    if (!tileCache || changeFlags.updateTriggersChanged) {
```

Comment:
@xintongxia This is unnecessarily re-downloading all the data when any update trigger changes. To invalid the cached layer instance, you should just set `tile.layer` to `null`.